stable/prometheus-rabbitmq-exporter/templates/deployment.yaml

```diff
@@ -46,6 +46,14 @@ spec:
               value: "{{ .Values.rabbitmq.skip_queues }}"
             - name: SKIP_VHOST
               value: "{{ .Values.rabbitmq.skip_vhost }}"
+            - name: RABBIT_EXPORTERS
+              value: "{{ .Values.rabbitmq.exporters }}"
+            - name: OUTPUT_FORMAT
+              value: "{{ .Values.rabbitmq.output_format }}"
+            - name: RABBIT_TIMEOUT
+              value: "{{ .Values.rabbitmq.timeout }}"
+            - name: MAX_QUEUES
+              value: "{{ .Values.rabbitmq.max_queues }}"
```

Comment:
In general, we want camel case, but since this matches the existing style, I'm going to accept this.